kinds/accesspolicy/access_policy_kind.cue

```diff
@@ -0,0 +1,33 @@
+package kind
+
+name:        "AccessPolicy"
+maturity:    "merged"
+description: "Access rules for a scope+role.  NOTE there is a unique constraint on role+scope"
+
+lineage: seqs: [
+	{
+		schemas: [
+			//0.0
+			{
+				// UID for user/team/service...
+				// ??? should this be a #ResourceID  but kind=user|team|???
+				role: string
```

Comment:
We'll need a Role kind as well, so that this is mapped to the Role, and the Role can then be linked to User/Team/BuiltinRole

Role
- namespace (role.org_id)
- uid (role.uid)
- id (role.name)
- title (role.display_name)
- group_name (role.group_name)
- description (role.description)
- version (role.version)
- hidden (role.hidden)

To create that association we'll likely also need kinds for UserRoleBinding, TeamRoleBinding, and BuiltinRoleBinding based on the user_role, team_role and builtin_role tables respectively

- namespace
- role
- user/team/builtin